### PR TITLE
[NP-6479] Add WizardInspector

### PR DESCRIPTION
**Pull request by KernelDeimos**
This PR adds a simple wizard inspector which will help with NP-6479 and future wizard improvements. This will also help to catch incorrect wizard behaviour much earlier.

To see the "Open Wizard Inspector" action you must have a capability with ID `developer`.
image
image

### CODE REVIEW

File: src/foam/u2/wizard/agents/DeveloperModeAgent.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/u2/wizard/agents/DeveloperModeAgent.js]: 2022

File: src/foam/u2/wizard/debug/WizardInspector.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
```
Comment (kgrgreer) [inline src/foam/u2/wizard/debug/WizardInspector.js]: 2022

File: src/foam/u2/wizard/debug/WizardInspector.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    'wizardController'
+  ],
+
+  requires: [
+    'foam.u2.DetailPropertyView',
+    'foam.u2.borders.Block'
+  ],
+
+  static: [
+    // ???: Should this and TranslationConsole extend a common class?
```
Comment (kgrgreer) [inline src/foam/u2/wizard/debug/WizardInspector.js:22]: No, not yet.

File: src/foam/u2/wizard/agents/DeveloperModeAgent.js (hunk 1 above)
Comment (KernelDeimos) [reply src/foam/u2/wizard/agents/DeveloperModeAgent.js]: Things like this really hurt when I'm not on the same branch anymore

**Pull request merged**